logits_processor_zoo/utils.py

```diff
@@ -26,3 +26,11 @@ def text_to_token(tokenizer: PreTrainedTokenizer, text: str, last: bool):
         raise Exception(f"Can't convert {text} to token. It has {len(tokens)} tokens.")
 
     return tokens[-1]
+
+
+def get_new_line_tokens(tokenizer):
+    new_line_tokens = set()
```

Comment:
You could use 
`set([idx for token, idx in tokenizer.get_vocab().items() if "\n" in tokenizer.decode(idx)])`
to get all tokens that contain "\n".
There are some other tokens such as `'...\n\n'` (for `meta-llama/Llama-3.2-3B-Instruct`) that might occur in theory, although I do not expect this to happen in a real world example.